### PR TITLE
Fixed typo in xmrstak/backend/amd/amd_gpu/gpu.cpp

### PR DESCRIPTION
**Pull request by djfinch**
cod --> code

### CODE REVIEW

**Comment by psychocrypt** [comment]
thx for the fix

**Comment by psychocrypt** [comment]
please do a fix only for the spelling issue. The change with the semixolon is wrong.

**Comment by djfinch** [comment]
Yep, thanks. I figure it out later and this typo is reverted already.

File: xmrstak/backend/amd/amd_gpu/gpu.cpp [hunk 1]
```diff
@@ -478,7 +478,7 @@ size_t InitOpenCLGpu(cl_context opencl_ctx, GpuContext* ctx, const char* source_
 			p_id++;
 		}
 
-		if( ret = clGetProgramInfo(ctx->Program, CL_PROGRAM_BINARIES, num_devices * sizeof(char*), all_programs.data(),NULL) != CL_SUCCESS)
+		if((ret = clGetProgramInfo(ctx->Program, CL_PROGRAM_BINARIES, num_devices * sizeof(char*), all_programs.data(),NULL)) != CL_SUCCESS)
```
Comment (fireice-uk) [inline xmrstak/backend/amd/amd_gpu/gpu.cpp]: Please leave the braces on this line alone they are needed if you set something and compare to it.

**Comment by psychocrypt** [comment]
This if condition is fixed in #1237

**Pull request closed without merging**